plugins/SSH.koplugin/main.lua

```diff
@@ -3,15 +3,18 @@ local InfoMessage = require("ui/widget/infomessage")  -- luacheck:ignore
 local InputDialog = require("ui/widget/inputdialog")
 local UIManager = require("ui/uimanager")
 local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
 local util = require("util")
 local _ = require("gettext")
 
---This plugin use a patched dropbear that add two things :
---  the -n option to allow login without password
---  read the keyfile from settings/SSH/authorized_keys
+-- This plugin use a patched dropbear that add two things :
+-- the -n option to allow login without password
+-- read the keyfile from settings/SSH/authorized_keys
 
-if not util.pathExists("dropbearmulti") then return { disabled = true, } end
+if not util.pathExists("dropbearmulti") then
+    return { disabled = true, }
```

Comment:
Although unless you write it as follows the smaller diff thing still doesn't apply. ;-)
```lua
    return {
        disabled = true,
    }
```

Reply:
Alright.